validate_sptx/test_experiment.py

```diff
@@ -30,3 +33,17 @@ def test_dartfish_example_experiment():
     dartfish_example = resource_filename(
         "validate_sptx", "examples/experiment/dartfish_experiment.json")
     assert validator.validate_file(dartfish_example)
+
+
+# see #614
+def test_no_manifest_example_experiment():
+    no_manifest_example = resource_filename(
+        "validate_sptx", "examples/experiment/no_manifest.json")
+
+    # SpaceTxValidator doesn't handle multiple files so this passes
```

Comment:
How much work do you think it would be to make SpaceTxValidator validate like the tree walker?

Reply:
Which I could see doing here as well. In the schema, I'd like to have a marker, though, that a string field points to a file and of which schema.